Doc/library/asyncio-stream.rst

```diff
@@ -88,12 +67,8 @@ and work with streams:
 
       The *ssl_handshake_timeout* parameter.
 
-   .. deprecated-removed:: 3.8 3.10
-
-      `open_connection()` is deprecated in favor of :func:`connect`.
-
 .. coroutinefunction:: start_server(client_connected_cb, host=None, \
-                          port=None, \*, loop=None, limit=2**16, \
+                          port=None, \*, loop=None, limit=None, \
```

Comment:
Can the change to the default value of *limit* be retained? I wasn't sure about this one, so I figured it would be best to ask.